querybook/webapp/lib/utils/cron.ts

```diff
@@ -136,24 +159,46 @@ export function getRecurrenceLocalTimeString(
         .format(format);
 }
 
+/**
+ * Determines whether a cron string is supported by the recurrence editor.
+ *
+ * @param cron Cron string to validate
+ * @returns true if cron string is supported by recurrence editor, false otherwise
+ */
```

Comment:
maybe we can also use [cron-parser](https://www.npmjs.com/package/cron-parser) for validation, which is mentioned by package `cronstrue`

Reply:
The backend has cron validation that prevents saving invalid crons, so I don't think we would need this here.  This function is only used to determine if the RecurrenceEditor should use `cron` mode or one of the simple modes.